.github/workflows/tests.yml

```diff
@@ -0,0 +1,52 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - master
+      - '*.x'
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  tests:
+    runs-on: ubuntu-22.04
+
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [7.4, '8.0', 8.1, 8.2]
```

Comment:
I only tested from 7.4 and up since that was the lowest version in Travis.

Reply:
We could try to add 7.3 and 7.2 to the matrix though and keep them if they run green.

Reply:
Seems to work!